doc/modules/cassandra/pages/managing/operating/compression.adoc

```diff
@@ -145,6 +140,17 @@ immediately, the operator can trigger an SSTable rewrite using
 `nodetool scrub` or `nodetool upgradesstables -a`, both of which will
 rebuild the SSTables on disk, re-compressing the data in the process.
 
+== Other options
+
+* `crc_check_chance` (default: `1.0`): determines how likely Cassandra
+is to verify the checksum on each compression chunk during reads to
+protect against data corruption. Unless you have profiles indicating
+this is a performance problem it is highly encouraged not to turn this
+off as it is Cassandra's only protection against bitrot. This option was part
```

Comment:
How about "In earlier versions of Cassandra a duplicate of this option existed in the compression configuration. The latter was deprecated in Cassandra 3.0 and removed in Cassandra 5.0."?

Reply:
sure